src/App.tsx

```diff
@@ -14,11 +14,80 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGoods: ['Jam'],
+  };
+
+  render() {
+    const { selectedGoods } = this.state;
+
+    return (
+      <div className="App">
+        <h1 className="App__title">
+          {selectedGoods.length === 0
+            ? 'No goods selected'
+            : (
+              <>
+                {`Selected Goods: ${selectedGoods.join(', ')}`}
+                <button
+                  type="button"
+                  className="App__clearButton"
+                  onClick={() => {
+                    this.setState({ selectedGoods: [] });
+                  }}
+                >
+                  Clear
+                </button>
```

Comment:
If one good is selected, there is should be the message:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/52370890/178721936-e116e83a-c56f-456b-a962-45adae0d7cdd.png">
 
Create a separate function for forming this message